### ISSUE TITLE
Cannot load saved models

### ISSUE THREAD

**Issue by zoejyl** [mljar/mljar-supervised#480]
I'm sorry if this question is trivial. I tried to load one of the saved models (e.g. learner_fold_0.catboost ) in order to get some insights into the model. I tried to import as json but it didn't work. I'm wondering which encoding the models used.

**Comment by pplonski** [comment]
Hi, Please try to load this model with the catboost package. The code used in AutoML:
https://github.com/mljar/mljar-supervised/blob/7bd8324e904a3dfa5d60bbebfbb1460d2672102c/supervised/algorithms/catboost.py#L301-L312

**Comment by pplonski** [comment]
I think it will be nice to add it to docs.

**Comment by dvirginz** [comment]
I agree. There is no documentation on saving and loading the models.
As it is an AutoML framework, I suggest a generic "one-solution fit all models" schema. I.e., loading models shouldn't be done manually; instead, calling the load model on the *folder* of the training should load the best model for serving.

Thank you!

**Comment by williamty** [comment]
As it is an AutoML framework, loading models shouldn't be done manually.

**Comment by pplonski** [comment]
Done, it is here https://supervised.mljar.com/features/save-and-load-models/
